build_and_wrap.py

```
@@ -185,6 +210,67 @@ def _locate_adapter_weights(adapter_dir: Path) -> Path | None:
+        raise FileNotFoundError(f"Merged model directory does not exist: {merged_dir}")
+
+    if output_dir.exists():
+        shutil.rmtree(output_dir)
+    ensure_directory(output_dir)
+
+    try:
+        from autoawq import AutoAWQForCausalLM  # type: ignore[import-not-found]
+    except ModuleNotFoundError as exc:  # pragma: no cover - dependency guard
+        raise RuntimeError(
+            "autoawq must be installed to export AWQ quantized models"
+        ) from exc
```

Ruff (0.13.3)

Avoid specifying long messages outside the exception class

(TRY003)